lib/database/utils.js

```diff
@@ -67,6 +67,10 @@ exports.promptAdminUser = function (callback) {
 }
 
 exports.checkCouchCredentials = function (env_config, callback) {
+  if (env_config.username && env_config.password) {
+    exports.credentials.set(env_config.hoodie.data_path, env_config.username, env_config.password);
```

Comment:
The reason the tests are failing so quickly is because of this semicolon on line 70. Hoodie-Server adheres to this [JavaScript Standard](https://github.com/feross/standard) which doesn't permit the use of semicolons. Would you mind remove the semicolon here and see if the tests start to pass when you push the changes? Thank you! :)